web/rest/client/features/my/call_forward_settings/getCallForwardSettings.feature

```diff
@@ -1,6 +1,6 @@
 Feature: Retrieve call forward settings
   In order to manage call forward settings
-  As an super admin
+  As a user
```

Comment:
As a**n** user?

Reply:
Not aparently https://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user

Reply:
Cool, it sound kinda weird tbh 😂